jxl/src/frame/color_correlation_map.rs

```diff
@@ -12,6 +12,8 @@ use std::default::Default;
 
 pub const COLOR_TILE_DIM: usize = 64;
 
+const _: () = assert!(COLOR_TILE_DIM % BLOCK_DIM == 0);
```

Comment:
Would it make any sense at all to `#[cfg(test)]` this so that it’s skipped in the final library / binaries? (Or is that not how it works?)

Reply:
I don't believe `const`s actually exist as far as the final binary is concerned :-)

Reply:
Ah, right :D I must have read it with my C++ glasses.